src/resources/learningResource/learningResource.model.js

```diff
@@ -0,0 +1,35 @@
+import mongoose from 'mongoose';
+
+const learningResourceSchema = new mongoose.Schema({
+  category: {
+    type: String,
+    enum: ['Coding', 'Maths', 'Electronics'],
+    required: true
+  },
+  minAge: {
+    type: Number,
+    required: true
+  },
+  maxAge: {
+    type: Number,
+    required: true
+  },
+  title: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  rating: {
+    type: Number,
+    min: 0,
+    max: 5,
+    required: true
+  },
+  body: String,
```

Comment:
Might make sense to require a description in the body. What do you think?

Reply:
I think as description field is optional in the front end form.

Reply:
It definitely feels like requiring a description would be a good idea here. I was imagining somebody might want to draft a card and add details later, though perhaps this is not intended functionality.

I see that the description field is optional, so for parity with the design I think I'll leave this for now.

Reply:
Yep, agreed keep it as per the design.